lib/services/livesync/livesync-service-base.ts

```diff
@@ -176,53 +175,45 @@ class LiveSyncServiceBase implements ILiveSyncServiceBase {
 			return (() => {
 				let shouldRefreshApplication = true;
 				let deviceAppData = this.$deviceAppDataFactory.create(appIdentifier, this.$mobileHelper.normalizePlatformName(platform), device, liveSyncOptions);
-				if (deviceAppData.isLiveSyncSupported().wait()) {
```

Comment:
isLiveSyncSupported is implemented only in AppBuilder so that's why I've removed it